go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/gammazero/deque v0.1.2
 	github.com/go-logr/logr v1.4.1
+	github.com/go-ping/ping v1.1.0
```

Comment:
There is a disclaimer in the README that this library is no longer maintained and is deprecated, so we should not be using it

Reply:
I willl change it.

Reply:
should this be removed now?

Reply:
I have tidied up libs.